src/views/v2/components/events-bar/views/list/item.php

```diff
@@ -28,7 +28,12 @@
 		href="<?php echo esc_url( $public_view_data->view_url ); ?>"
 		class="tribe-events-c-view-selector__list-item-link"
 		data-js="tribe-events-view-link"
-		aria-label="<?php echo esc_attr__( 'View events in ', 'the-events-calendar' ) . esc_attr( $public_view_data->view_label ) . ' ' . esc_attr__( 'View', 'the-events-calendar' ); ?>"
+		aria-label="
+		<?php
+		/* translators: %1$s: Events plural text, %2$s: View label */
+		printf( esc_attr__( 'View %1$s in %2$s View', 'the-events-calendar' ), esc_html( tribe_get_event_label_plural() ), esc_html( $public_view_data->view_label ) ); 
```

Comment:
```suggestion
		printf( esc_attr__( 'Display %1$s in %2$s View', 'the-events-calendar' ), esc_html( tribe_get_event_label_plural() ), esc_html( $public_view_data->view_label ) ); 
```

Less repetitive/redundant